addons/sale_crm/models/crm_lead.py

```diff
@@ -60,12 +50,8 @@ def action_new_quotation(self):
 
     def action_view_sale_quotation(self):
         action = self.env["ir.actions.actions"]._for_xml_id("sale.action_quotations_with_onboarding")
-        action['context'] = {
-            'search_default_draft': 1,
-            'search_default_partner_id': self.partner_id.id,
-            'default_partner_id': self.partner_id.id,
-            'default_opportunity_id': self.id
-        }
+        action['context'] = self._get_opportunity_quotation_context()
+        action['context'].update({'search_default_draft': 1})
```

Comment:
When it's a single value you can use the key directly.

`action['context']['search_default_draft'] = 1`

(Same remark for here above)